src/pages/BO/dashboard/index.ts

```diff
@@ -0,0 +1,11 @@
+import semver from "semver";
+import BOBasePage from "../BOBasePage";
+
+let dashboardPage: BOBasePage;
```

Comment:
@boubkerbribri I have an idea : we can create an interface for each page that we apply to each version. Same interface, same methods. What do you think about that ?